src/main/java/minicraft/screen/QuestsDisplay.java

```diff
@@ -0,0 +1,342 @@
+		return new ArrayList<>(quests.values());
+	}
+	public static ArrayList<Quest> getUnlockedQuests() {
+		return unlockedQuests;
+	}
+	public static ArrayList<Quest> getCompleteQuest() {
+		return completeQuest;
+	}
+	public static HashMap<String, QuestStatus> getStatusQuests() {
+		return questStatus;
+	}
+
+	public static boolean isQuestDone(String name) {
+		return completeQuest.contains(getQuest(name));
+	}
+
+	public static Object getQuestData(String name) {
+		return questStatus.get(name);
+	}
+	public static void setQuestData(String name, QuestStatus val) {
+		questStatus.put(name, val);
+	}
+	public static void setQuestDataVal(String name, Object val) {
+		questStatus.get(name).val = val;
+	}
+
+	public static void unlockQuest(String name) {
+		Quest quest = quests.get(name);
+		if (quest == null) return;
+		if (unlockedQuests.contains(quest)) return;
+
+		quest.unlock();
+		unlockedQuests.add(quest);
+		Game.notifications.add(Localization.getLocalized("minicraft.notification.quest_unlocked") + " " + Localization.getLocalized(name));
+	}
+	public static void completeQuest(String name) { completeQuest(name, true); }
+	public static void completeQuest(String name, boolean mustUnlocked) {
+		Quest quest = quests.get(name);
+		if (quest == null) return;
+		if (completeQuest.contains(quest)) return;
+		if (mustUnlocked && !quest.getUnlocked()) return;
+
+		completeQuest.add(quest);
+		Game.notifications.add(Localization.getLocalized("minicraft.notification.quest_done") + " " + Localization.getLocalized(name));
+		for (String q : quest.getUnlocks()) unlockQuest(q);
+	}
+
+	public static void resetGameQuests() {
+		unlockedQuests.clear();
+		completeQuest.clear();
+		questStatus.clear();
+
+		for (Quest quest : quests.values()) {
+			if (initiallyUnlocked.contains(quest.id)) {
+				quest.unlock();
+				unlockedQuests.add(quest);
+			} else {
+				quest.lock();
+			}
+		}
+	}
+
+	public static void loadGameQuests(ArrayList<String> unlocked, ArrayList<String> done) { loadGameQuests(unlocked, done, new HashMap<>()); }
+	public static void loadGameQuests(ArrayList<String> unlocked, ArrayList<String> done, Map<String, String> data) {
+		resetGameQuests();
+
+		for (String n : unlocked) {
+			Quest q = getQuest(n);
+			if (!unlockedQuests.contains(q))
+				unlockedQuests.add(q);
+		}
+		for (String n : done) completeQuest.add(getQuest(n));
+		for (Entry<String, String> e : data.entrySet()) questStatus.put(e.getKey(), QuestStatus.fromStringType(e.getValue()));
+	}
+
+	@Override
+	public void tick(InputHandler input) {
+		if (entrySelected) {
+			if (input.getKey("exit").clicked) {
+				menus[0].shouldRender = true;
+				entrySelected = false;
+			}
+			return;
+		}
+
+		super.tick(input);
+
+		if (input.getKey("cursor-left").clicked) if (selectedEntry > 0) {
+			selectedEntry--;
+			updateEntries();
+		};
+		if (input.getKey("cursor-right").clicked) if (selectedEntry < 1) {
+			selectedEntry++;
+			updateEntries();
+		};
```

Comment:
Closing brackets don't need semicolons.

Add newlines after the closing brackets.